samples/bluetooth/alexa_gadget/src/gadgets/CMakeLists.txt

```
@@ -0,0 +1,181 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# The Gadgets Profile relies on the nanopb generator tool, which must be
+# available at the configuration stage.
+# This tool is used to compile the alexa .proto files at build time.
+# Output from the .proto compilation is a set of .c and .h files.
+
+# Note: as there doesn't seem to be an easy way of checking the version of the nanopb generator (protoc-gen-nanopb binary)
+# there is the possibility of a version mismatch between the generator and the nanopb source files that is used in the firmware sample.
+# If there is a mismatch, there will be a compilation error in the build step.
+# The easiest way to fix such an error is to download the appropriate nanopb generator .zip and extract it to this sample folder.
+# (https://jpa.kapsi.fi/nanopb/download/)
+
+
+# Check system for protoc and nanopb binaries presence.
+# Prefer local variable to make version matching easier
+find_program(PROTOC_BIN protoc HINTS "${PROJECT_SOURCE_DIR}/*/generator-bin/")
+find_program(NANOPB_BIN protoc-gen-nanopb HINTS "${PROJECT_SOURCE_DIR}/*/generator-bin/")
+
+if ((NOT NANOPB_BIN) OR (NOT PROTOC_BIN))
+    message(STATUS "nanopb not found.")
+    message(STATUS "Please download platform-specific nanopb from here https://jpa.kapsi.fi/nanopb/download/")
+    message(STATUS "and extract to either this sample directory or somewhere cmake can find it via system PATH.")
```

How about:
```suggestion
    message(STATUS "Extract the files in `${PROJECT_SOURCE_DIR}`, or add `<path-to>/nanopb-x.y.z-<platform>` to system PATH.")
    message(STATUS "Alternatively, specify install location using: `-DCMAKE_PROGRAM_PATH=<path-to>/nanopb-x.y.z-<platform>/generator-bin` when runing CMake.")
```

So the user knows that he can actually install `nanopb` anywhere, and not need to change `PATH`

---

Yes, this seems fine to me